### ISSUE TITLE
If swal is handed an array of strings for its second parameter, it tries to call split on the array and errors out.

### ISSUE THREAD

**Issue by daerogami** [limonte/sweetalert2#345]
Another pattern I've noticed with the older swal is that you could hand it an array of strings. Since sweet alert 2 calls split on what it assumes is always a string, it results in the following error
```
angular.js:12416 TypeError: params.text.split is not a function
    at setParameters (sweetalert2.js:517)
    at modalDependant (sweetalert2.js:805)
    at sweetAlert (sweetalert2.js:1398)
    at responseError (angular-error-handling.js:43)
    at processQueue (angular.js:14634)
    at angular.js:14650
    at Scope.$eval (angular.js:15916)
    at Scope.$digest (angular.js:15727)
    at Scope.$apply (angular.js:16024)
    at done (angular.js:10511)
```
the call that causes this looks like this
`swal("Error", methodThatReturnsArrayOfStrings(objectWithErrorInfo), "error");`
it is easily corrected by calling `.join("\n")` on the return value but I didn't know if you wanted to catch old patterns.

**Comment by birjolaxew** [comment]
Seems like a general issue with stringifying the given content. SweetAlert (the old one) uses the built-in type coercion to turn input into text (eg `["foo", "bar"]` becomes `"foo,bar"` and `{foo: "bar"}` becomes `[object Object]`).

Whether we want to implement this is up to @limonte, but it's as easy as doing `"" + <value>` if we expect a string. If implemented, it should also be extended to other options that expect arbitrary strings (eg. `title` and `html`)

**Comment by limonte** [comment]
Fixed in [v6.0.1](https://github.com/limonte/sweetalert2/releases/tag/v6.0.1)
